docs/websites/lamp/lamp-server-on-ubuntu-14-04-trusty.md

```diff
@@ -0,0 +1,298 @@
+
+3. At this point MySQL should be ready to configure and run. While you shouldn't need to change the configuration file, note that it is located at `/etc/mysql/my.cnf` for future reference. The default values should be fine for a **Linode 1GB**, but if you decide to adjust them you should first make a backup copy:
+
+        sudo cp /etc/mysql/my.cnf ~/my.cnf.backup
+
+### Configure MySQL and Set Up MySQL databases
+ 
+ 1. After installing MySQL, it’s recommended that you run `mysql_secure_installation`, a program that helps secure MySQL. While running `mysql_secure_installation`, you will be presented with the opportunity to change the MySQL root password, remove anonymous user accounts, disable root logins outside of localhost, and remove test databases and reload privileges. It is recommended that you answer yes to these options but there is no obligation. Run the following command to execute the program:
+
+        mysql_secure_installation
+
+2. Next, we'll create a database and grant your users permissions to use databases. First, log in to MySQL:
+
+        sudo mysql -u root -p
+ 
+3. To create a new database and grant your users permissions on it, issue the following commands. Note, the semi-colons (`;`) at the end of the lines are crucial for ending the commands. Your commands should look similar to the following:
+
+        create database example_database_name;
+        grant all on example_database_name.* to 'example_user'@'localhost' identified by 'example_password';
+
+4. Note that database user names and passwords are only used by scripts connecting to the database, and that database user account names need not (and perhaps should not) represent actual user accounts on the system. If you need to create additional users in the database you just created, simply run the command below, substituting the new user name and password where appropriate:
+
+        grant all on example_database_name.* to 'example_user2'@'localhost' identified by 'example_password2';
+
+5. With that completed, you’ve successfully configured MySQL. To exit the MySQL database administration utility issue the following command:
+
+        quit
+
+With Apache and MySQL installed you are now ready to move on to installing PHP to provide scripting support for your web pages.
+
+
+
+
+Intall PHP5, PHP-FPM and Configuring them for use with Apache
+-------------------------------
+PHP makes it possible to produce dynamic and interactive pages using your own scripts and popular web development frameworks. Furthermore, many popular web applications like WordPress are written in PHP. If you want to be able to develop your websites using PHP, you must first install it. 
+
+1. Ubuntu includes packages for installing PHP5 with support for PHP5-FPM. From the terminal, issue the following command:
+
+        sudo apt install libapache2-mod-fastcgi php5-fpm php5
```

Comment:
"mod_fastcgi is a relic from the httpd 1.3 days that is no longer maintained. In httpd 2.2, these APIs were removed, "

So this is why we definitely do not need the instructions for the PHP-FPM in the config instructions. Ok I will remove that as well.